### ISSUE TITLE
Personal Access Token Table not present in latest release

### ISSUE THREAD

**Issue by kunalbedi** [Blair2004/NexoPOS#1757]
**Describe the bug**
Personal Access Token Table not present in latest release

**To Reproduce**
1. Go to 'User Profile', Token Tab
2. Error shown that "Unable to load token, Un unexpected error has happened"

**Expected behavior**
Personal Access Token Table should be created at setup and the admin user or any user created should have the token created for him when created.

**Environment:**
 - NexoPOS 5.0.2
 - Fedora  Linux with LAMP environment
 - PHP 8.2.15

**Comment by Blair2004** [comment]
Thank you for reporting. We're checking

**Comment by Blair2004** [comment]
![image](https://github.com/Blair2004/NexoPOS/assets/5265663/56eb66fd-32c4-403b-984a-e68c0b33c1ee)
We've used v4.nexopos.com, which pulls the latest stable release every hour and creates a demo from it. We checked the "Profile" and  "API Token" tab, and it seems to work perfectly. We might need further details on the context to reproduce it.

Meanwhile, the thread is closed, feel free to open it with further information.
